test/lib/specHelperSpec.js

```diff
@@ -95,13 +101,18 @@ describe('Fluxxed up test helpers', function() {
       // Manipulate the DOM:
       rig.fillIn('input', 'new thing')
       rig.clickLink('Submit')
+
       rig.toggleCheckbox('#checkbox-input')
+      rig.toggleRadioButton('.radio-input')
+      rig.setValue('.select-input', '1')
 
       // Set the expectations:
       // (TODO: make these Chai DSL.)
       rig.setExpectationCallback(() => {
         expect(rig.domNode.find('.answer').text()).to.match(/new thing$/)
         expect(rig.domNode.find('#checkbox-input')[0].checked).to.match(/true/)
+        expect(rig.domNode.find('.radio-input')[0].checked).to.equal(true)
```

Comment:
we're sort of missing the point of the test here.  If you notice the button click cause the form values to be put in the '.answer' element. The test as written doesn't care about the inner state, it still checking the dom. Take a closer look please.